modules/ROOT/pages/salesforce/salesforce-connector-100-upgrade-migrate.adoc

```diff
@@ -0,0 +1,228 @@
+= Salesforce - Upgrade and Migrate - Mule 4
```

Comment:
NEW EDIT BEGINS HERE

Salesforce Connector 10 Upgrade and Migrate: https://github.com/mulesoft/docs-connectors/pull/621
This is a topic I wrote recently.  Hopefully, this one is also in good shape. I do get confused about when to use back ticks and when not to, but I think this is something that you and Mysti are working on.